rclpy/test/test_node.py

```diff
@@ -175,6 +177,70 @@ def test_set_executor_clear_executor(self):
         finally:
             node.destroy_node()
 
+    def test_node_set_parameters(self):
```

Comment:
Based on feedback from https://github.com/ros2/rclpy/pull/216#discussion_r208750095 it looks like we now prefer using directly the assert statement rather than the ones provided by unittest (same multiple times below). Though not a blocker for this PR IMO